python/graphstorm/config/argument.py

```diff
@@ -578,6 +579,19 @@ def model_encoder_type(self):
             f"Model encoder type should be in {BUILTIN_ENCODER}"
         return self._model_encoder_type
 
+    @property
+    def input_activate(self):
+        """ Whether include activation layer and dropout in the input layer
+        """
+        # pylint: disable=no-member
+        if hasattr(self, "_input_activate"):
+            assert self._input_activate in [True, False], \
+                "Use input activate flag must be True or False"
+            return self._input_activate
+
+        # By default, use mini batch inference, which requires less memory
+        return True
```

Comment:
return None by default.

Reply:
I think here we should return relu as the default option.